rust/processor/src/db/common/models/emojicoin_models/tests.rs

```diff
@@ -236,9 +236,10 @@ mod json_tests {
                 "0xd00db145c047cd3619ecba69e45b4ad77f43737d309d8113d6c1c35f7a8dd00d"
             );
             assert_eq!(e.integrator_fee, 100000000);
-            assert_eq!(e.market_metadata.emoji_bytes, [
-                240, 159, 152, 141, 240, 159, 152, 156
-            ]);
+            assert_eq!(
+                e.market_metadata.emoji_bytes,
+                [240, 159, 152, 141, 240, 159, 152, 156]
+            );
```

Comment:
```suggestion
            assert_eq!(e.market_metadata.emoji_bytes, [
                240, 159, 152, 141, 240, 159, 152, 156
            ]);
```

Reply:
I'm not sure how your changes snuck in, but my format on save feature reverts them to what the were before, which was already passing CI as I understand

Reply:
Updated my toolchain to nightly in vscode as well so save on format is the correct formatter, should be good from now on!